web/docs/tutorials/todo-app/06-auth.md

```diff
@@ -135,11 +135,11 @@ export default SignupPage
 ```
 
 
-## Updating `MainPage` page to check if user is authenticated
+## Updating `MainPage` page to check if the user is authenticated
 
-Now, let's see how we're going to handle the situation when user is not logged in.
+Now, let's see how we're going to handle the situation when the user is not logged in.
 `MainPage` page is a private page and we want users to be able to see it only if they are authenticated.
-There is a specific Wasp feature that allows us to achieve this in a simple way:
+There is a specific Wasp feature that allows us to achieve this simply:
```

Comment:
"In a simple way" is more clear than "simply" in this case. Putting the adverb "simply" at the end of the sentence seems unnatural, and putting it in the middle emphasizes the wrong point.
```suggestion
Wasp allows you to simply enforce private pages using the `authRequired` field:
```
Or something like that.